src/containers/category-dropdown/CategoryDropdown.tsx

```diff
@@ -81,9 +85,9 @@ const CategoryDropdown = ({
         })
       })
 
-      setIsFetched(false)
+      void fetchAllCategoriesNames()
```

Comment:
what does this line mean?

Reply:
`setIsFetched` was responsible for indicating whether it needed to fetch categories. When categories were fetched, this state was changed to true. When new category was created, it switched to false (that is the line you selected). There was this logic, because categories were fetched on focus. This state existed so categories wouldn't be fetched on every focus, but only on those before which new category was created (and there were no fetches after category was created). I slightly changed this logic to fetch categories each time after new category is created (I added call to do this in line you selected), just inside `onResponse` callback. I think if user creates new category inside of this dropdown, he will definitely want it to appear in this list, so there is no sense to wait for focus on dropdown.

Reply:
Thank you so much for this explanation, very informative, totally agree with your point! 
But i was curios `void fetchAllCategoriesNames()` about this line, never saw `void` before function call

Reply:
I did it to overcome lint error that occurs when trying to call func that returns promise without await: `Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the void operator  @typescript-eslint/no-floating-promises`. Since Bohdan also mentioned it, I will try to overcome it another way.